pkg/manager/impl/execution_manager.go

```diff
@@ -1346,3 +1356,25 @@ func NewExecutionManager(
 		qualityOfServiceAllocator: executions.NewQualityOfServiceAllocator(config, resourceManager),
 	}
 }
+
+func (m *ExecutionManager) addProjectLabels(ctx context.Context, projectName string, initialLabels map[string]string) (map[string]string, error) {
```

Comment:
I know you have a comment below, but comments on the func become docs in golang.
So a comment like
// Adds project labels with higher precedence to workflow labels. Project labels are ignored if a corresponding label is set on the workflow.

Reply:
@kumare3 I addressed the comment